File: AMS.API/DTO/SkillCouncil/SkillCouncilResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace AMS.API.DTO.SkillCouncil
{
    public class SkillCouncilResponse
    {
        #region Constructor

        public SkillCouncilResponse()
        {
            SkillCouncil = new List<SkillCouncil>();
            OperationStatus = new OperationStatus();
        }
        #endregion Constructor

        #region Class Properties

        public List<SkillCouncil> SkillCouncil { get; set; }

        public OperationStatus OperationStatus { get; set; }

        #endregion
    }
}

File: AMS.Web/src/app/models/manage-skill-council/skill-council-response.ts
import { SkillCouncil } from './skill-council';
import { OperationStatus } from '../shared/operation-status';

export class SkillCouncilResponse
{
    //#region Constructor

    constructor()
    {
        this.SkillCouncil = new Array<SkillCouncil>();
        this.OperationStatus = new OperationStatus();
    }

    //#endregion Constructor

    //#region Class Properties

    SkillCouncil: Array<SkillCouncil>;

    OperationStatus: OperationStatus;

    //#endregion
}